integration_tests/grpc_server/test_docker_grpc_server.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import os
+from typing import Text, List, Tuple
+
+import grpc
+import pytest
+
+from google.protobuf.json_format import MessageToDict
+from grpc_health.v1 import health_pb2
+from grpc_health.v1.health_pb2_grpc import HealthStub
+
+from rasa_sdk.grpc_py import action_webhook_pb2_grpc, action_webhook_pb2
+from rasa_sdk.grpc_server import GRPC_ACTION_SERVER_NAME
+from integration_tests.conftest import ca_cert, client_key, client_cert
+
+GRPC_HOST = os.getenv("GRPC_ACTION_SERVER_HOST", "localhost")
+GRPC_NO_TLS_PORT = os.getenv("GRPC_NO_TLS_PORT", 7010)
+GRPC_TLS_HOST = os.getenv("GRPC_ACTION_SERVER_TLS_HOST", "localhost")
+GRPC_TLS_PORT = os.getenv("GRPC_TLS_PORT", 7011)
+GRPC_REQUEST_TIMEOUT_IN_SECONDS = os.getenv("GRPC_REQUEST_TIMEOUT_IN_SECONDS", 5)
+
+logger = logging.getLogger(__name__)
```

Comment:
Should `structlog` be used here as well?

Reply:
Rasa SDK does not use `structlog` ATM. We need a separate line of work for that.